[0m16:17:42.562653 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7e50e70e55b0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7e50e93c2420>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7e50e96d1a60>]}


============================== 16:17:42.565034 | 2eb4a03f-2a4b-4b6f-bb9a-ba568e0f2cd1 ==============================
[0m16:17:42.565034 [info ] [MainThread]: Running with dbt=1.8.7
[0m16:17:42.565496 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': '/home/pablo/.dbt', 'fail_fast': 'False', 'warn_error': 'None', 'log_path': 'logs', 'version_check': 'True', 'debug': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'None', 'quiet': 'False', 'no_print': 'None', 'log_format': 'default', 'introspect': 'True', 'static_parser': 'True', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'target_path': 'None', 'invocation_command': 'dbt init .', 'send_anonymous_usage_stats': 'True'}
[0m16:17:42.571915 [warn ] [MainThread]: [ConfigFolderDirectory]: Unable to parse logging event dictionary. Failed to parse dir field: expected string or bytes-like object, got 'PosixPath'.. Dictionary: {'dir': PosixPath('/home/pablo/.dbt')}
[0m16:17:42.572238 [info ] [MainThread]: Creating dbt configuration folder at 
[0m16:18:01.787544 [debug] [MainThread]: Starter project path: /home/pablo/repos/pruna_technical_test/.venv/lib/python3.12/site-packages/dbt/include/starter_project
[0m16:18:01.790190 [info ] [MainThread]: 
Your new dbt project "f1_ergast_api" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m16:18:01.790556 [info ] [MainThread]: Setting up your profile.
[0m16:18:19.958299 [info ] [MainThread]: No sample profile found for clickhouse.
[0m16:18:19.959498 [debug] [MainThread]: Resource report: {"command_name": "init", "command_success": true, "command_wall_clock_time": 37.433254, "process_user_time": 0.978968, "process_kernel_time": 0.108774, "process_mem_max_rss": "82792", "process_in_blocks": "37245", "process_out_blocks": "12"}
[0m16:18:19.960015 [debug] [MainThread]: Command `dbt init` succeeded at 16:18:19.959932 after 37.43 seconds
[0m16:18:19.960344 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7e50e93c2420>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7e50e70e7680>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7e50e7cb1460>]}
[0m16:18:19.960698 [debug] [MainThread]: Flushing usage events
[0m16:26:23.657101 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7a7c061b8fb0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7a7c05f70a70>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7a7c06173c20>]}


============================== 16:26:23.659314 | ff01b0bd-3eec-4a53-be85-5d5e42e2e07e ==============================
[0m16:26:23.659314 [info ] [MainThread]: Running with dbt=1.8.7
[0m16:26:23.659826 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'version_check': 'True', 'profiles_dir': '/home/pablo/.dbt', 'log_path': 'logs', 'debug': 'False', 'fail_fast': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'None', 'quiet': 'False', 'no_print': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'invocation_command': 'dbt debug', 'log_format': 'default', 'static_parser': 'True', 'target_path': 'None', 'introspect': 'True', 'send_anonymous_usage_stats': 'True'}
[0m16:26:23.667062 [info ] [MainThread]: dbt version: 1.8.7
[0m16:26:23.667434 [info ] [MainThread]: python version: 3.12.3
[0m16:26:23.667845 [info ] [MainThread]: python path: /home/pablo/repos/pruna_technical_test/.venv/bin/python
[0m16:26:23.668197 [info ] [MainThread]: os info: Linux-6.8.0-45-generic-x86_64-with-glibc2.39
[0m16:26:23.668516 [info ] [MainThread]: Using profiles dir at /home/pablo/.dbt
[0m16:26:23.668851 [info ] [MainThread]: Using profiles.yml file at /home/pablo/.dbt/profiles.yml
[0m16:26:23.669170 [info ] [MainThread]: Using dbt_project.yml file at /home/pablo/repos/pruna_technical_test/dbt/dbt_project.yml
[0m16:26:23.669487 [info ] [MainThread]: Configuration:
[0m16:26:23.669844 [info ] [MainThread]:   profiles.yml file [[31mERROR not found[0m]
[0m16:26:23.670244 [info ] [MainThread]:   dbt_project.yml file [[31mERROR not found[0m]
[0m16:26:23.670598 [info ] [MainThread]: Required dependencies:
[0m16:26:23.670987 [debug] [MainThread]: Executing "git --help"
[0m16:26:23.673486 [debug] [MainThread]: STDOUT: "b"usage: git [-v | --version] [-h | --help] [-C <path>] [-c <name>=<value>]\n           [--exec-path[=<path>]] [--html-path] [--man-path] [--info-path]\n           [-p | --paginate | -P | --no-pager] [--no-replace-objects] [--bare]\n           [--git-dir=<path>] [--work-tree=<path>] [--namespace=<name>]\n           [--config-env=<name>=<envvar>] <command> [<args>]\n\nThese are common Git commands used in various situations:\n\nstart a working area (see also: git help tutorial)\n   clone     Clone a repository into a new directory\n   init      Create an empty Git repository or reinitialize an existing one\n\nwork on the current change (see also: git help everyday)\n   add       Add file contents to the index\n   mv        Move or rename a file, a directory, or a symlink\n   restore   Restore working tree files\n   rm        Remove files from the working tree and from the index\n\nexamine the history and state (see also: git help revisions)\n   bisect    Use binary search to find the commit that introduced a bug\n   diff      Show changes between commits, commit and working tree, etc\n   grep      Print lines matching a pattern\n   log       Show commit logs\n   show      Show various types of objects\n   status    Show the working tree status\n\ngrow, mark and tweak your common history\n   branch    List, create, or delete branches\n   commit    Record changes to the repository\n   merge     Join two or more development histories together\n   rebase    Reapply commits on top of another base tip\n   reset     Reset current HEAD to the specified state\n   switch    Switch branches\n   tag       Create, list, delete or verify a tag object signed with GPG\n\ncollaborate (see also: git help workflows)\n   fetch     Download objects and refs from another repository\n   pull      Fetch from and integrate with another repository or a local branch\n   push      Update remote refs along with associated objects\n\n'git help -a' and 'git help -g' list available subcommands and some\nconcept guides. See 'git help <command>' or 'git help <concept>'\nto read about a specific subcommand or concept.\nSee 'git help git' for an overview of the system.\n""
[0m16:26:23.673917 [debug] [MainThread]: STDERR: "b''"
[0m16:26:23.674191 [info ] [MainThread]:  - git [[32mOK found[0m]

[0m16:26:23.674479 [info ] [MainThread]: Connection test skipped since no profile was found
[0m16:26:23.674849 [info ] [MainThread]: [31m2 checks failed:[0m
[0m16:26:23.675173 [info ] [MainThread]: dbt looked for a profiles.yml file in /home/pablo/.dbt/profiles.yml, but did
not find one. For more information on configuring your profile, consult the
documentation:

https://docs.getdbt.com/docs/configure-your-profile


[0m16:26:23.675470 [info ] [MainThread]: Project loading failed for the following reason:
 project path </home/pablo/repos/pruna_technical_test/dbt/dbt_project.yml> not found

[0m16:26:23.676095 [debug] [MainThread]: Resource report: {"command_name": "debug", "command_wall_clock_time": 0.05707425, "process_user_time": 0.919638, "process_kernel_time": 0.095962, "process_mem_max_rss": "82472", "process_in_blocks": "35836", "process_out_blocks": "10", "command_success": false}
[0m16:26:23.676539 [debug] [MainThread]: Command `dbt debug` failed at 16:26:23.676461 after 0.06 seconds
[0m16:26:23.676908 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7a7c061b8fb0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7a7c06baa930>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7a7c081a1dc0>]}
[0m16:26:23.677242 [debug] [MainThread]: Flushing usage events
[0m16:55:17.573166 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7d70a4a44110>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7d70a401e330>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7d70a6fcab10>]}


============================== 16:55:17.575442 | c1aa4cdc-e219-47d8-bc17-0c64534df4f9 ==============================
[0m16:55:17.575442 [info ] [MainThread]: Running with dbt=1.8.7
[0m16:55:17.575968 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': '/home/pablo/.dbt', 'version_check': 'True', 'warn_error': 'None', 'log_path': 'logs', 'fail_fast': 'False', 'debug': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'static_parser': 'True', 'invocation_command': 'dbt test', 'log_format': 'default', 'target_path': 'None', 'introspect': 'True', 'send_anonymous_usage_stats': 'True'}
[0m16:55:17.576379 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path /home/pablo/repos/pruna_technical_test/dbt/dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m16:55:17.577057 [debug] [MainThread]: Resource report: {"command_name": "test", "command_wall_clock_time": 0.040736936, "process_user_time": 0.959616, "process_kernel_time": 0.136087, "process_mem_max_rss": "82048", "process_in_blocks": "36003", "process_out_blocks": "3", "command_success": false}
[0m16:55:17.577511 [debug] [MainThread]: Command `dbt test` failed at 16:55:17.577426 after 0.04 seconds
[0m16:55:17.577886 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7d70a3d51460>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7d70a43be5a0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x7d70a3f735f0>]}
[0m16:55:17.578222 [debug] [MainThread]: Flushing usage events
